src/app/(logged-in)/my/account/account-form.tsx

```diff
@@ -35,10 +37,7 @@ export default function AccountForm(props: AccountFormProps) {
 					<TextField label='Télefono de contacto' name='contactPhone' defaultValue={user.contactPhone ?? ''} validate={validate.contactPhone} className='grow mb-2'/>
 				</div>
 			</div>
-
-			<TextField label='Correo electrónico' name='email' validate={validate.email} defaultValue={user.email} className='mb-2'/>
-
+			{props.sessionType === 'google' ? null : <TextField label='Correo electrónico' name='email' validate={validate.email} defaultValue={user.email} className='mb-2'/>}
```

Comment:
Don't check whether Google is the current auth provider, instead invert the condition and check whether Auth0 is the auth provider